internal/cri/streamingserver/portforward/httpstream.go

```diff
@@ -43,15 +58,15 @@ func handleHTTPStreams(req *http.Request, w http.ResponseWriter, portForwarder P
 	}
 	streamChan := make(chan httpstream.Stream, 1)
 
-	klog.V(5).InfoS("Upgrading port forward response")
+	log.L.Info("Upgrading port forward response")
 	upgrader := spdy.NewResponseUpgrader()
 	conn := upgrader.UpgradeResponse(w, req, httpStreamReceived(streamChan))
 	if conn == nil {
 		return errors.New("unable to upgrade httpstream connection")
 	}
 	defer conn.Close()
 
-	klog.V(5).InfoS("Connection setting port forwarding streaming connection idle timeout", "connection", conn, "idleTimeout", idleTimeout)
+	log.L.Infof("Connection setting port forwarding streaming connection idle timeout connection idleTimeout %v", idleTimeout)
```

Comment:
(multiple occurrences) If we're switching to our logger, it'd be better to still retain the structured output.  But we can do this in a follow up PR if we want to merge more quickly.

```suggestion
	log.L.WithField("idleTimeout", idleTimeout).Info("Connection setting port forwarding streaming connection")
```

Reply:
fair.. will come back around to catch this in a follow up